src/main/java/bio/terra/cloudfiletodatastore/deltalayer/DeltaLayerBQJSONWriter.java

```diff
@@ -0,0 +1,75 @@
+package bio.terra.cloudfiletodatastore.deltalayer;
+
+import bio.terra.cloudfunctions.common.GsonWrapper;
+import com.google.cloud.bigquery.*;
+import com.google.common.annotations.VisibleForTesting;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Use a lower level API to write to BQ; this API is more succinct if there are no errors but has
+ * the disadvantage of quietly swallowing errors unless we inspect the {@link JobStatus}
+ */
+public class DeltaLayerBQJSONWriter implements DeltaLayerBigQueryWriter {
+
+  private static final Logger logger = Logger.getLogger(DeltaLayerBQJSONWriter.class.getName());
+
+  @Override
+  public void insertRows(
+      List<Map<String, Object>> inserts, String dataSet, String project, BigQuery bigQuery) {
+    WriteChannelConfiguration writeChannelConfiguration =
+        getWriteChannelConfiguration(dataSet, project);
+    for (Map<String, Object> insert : inserts) {
+      byte[] jsonBytes = getBytesForInsert(insert);
+      TableDataWriteChannel writer = bigQuery.writer(writeChannelConfiguration);
+      try {
+        writer.write(ByteBuffer.wrap(jsonBytes));
+        writer.close();
+      } catch (IOException e) {
+        logger.log(Level.SEVERE, "Could not write to BQ", e);
+        return;
+      }
```

Comment:
I think try with resources is out because we need the writer reference after close is called in order to do the error check (try with resources will cause the writer reference to disappear from scope after it's closed).  We could move the close to a finally block but it's pretty ugly..
```    
try {
        writer.write(ByteBuffer.wrap(jsonBytes));
      } catch (IOException e) {
        logger.log(Level.SEVERE, "Could not write to BQ", e);
        return;
      } finally {
        try {
          writer.close();
        } catch (IOException e) {
          logger.log(Level.SEVERE, "Could not close BQ writer", e);
        }
      }
```